components/Header.tsx

```diff
@@ -50,12 +46,7 @@ const Header = () => {
         </nav>
 
         <div className="switchers">
-          <button
-            className="dark-theme-switcher"
-            type="button"
-            title={toggleDarkMode}
-            aria-label={toggleDarkMode}
-          >
+          <button>
```

Comment:
Please re-read the issue behind this PR carefully.

The ask is to remove only the DarkMode Toggle from the Website Redesign, not the current living nodejs.org website.

Reply:
As you can see, the Issue (https://github.com/nodejs/nodejs.org/issues/5703) asks the removal of `DarkModeToggle` component. This file `Header.tsx` clearly doesn't use that component. So it should not be removed, nor the i18n translation messages related to it 👀

Reply:
I see that the issue has an error on the description, and it erroneously asked to remove the i18n translation message.

Reply:
I told you what to do. 👀 . Undo the unrelated changes to the `DarkModeToggle` component, such as deleting the i18n keys and removing the "dark mode toggle" from `Header.tsx` (not `NewHeader.tsx`)